tenntom/sink-repair
# Pioneer chaos

updated the list to sort automatically based on completion status.
To test:
Open in chrome
add a new request and ensure that it displays in the request list at or near the top.
choose a plumber for that request.
confirm that the request displays at or near the bottom of the list.
